db/migrate/20191009142101_add_institution_columns_to_institution_programs_archives.rb

```diff
@@ -0,0 +1,5 @@
+class AddInstitutionColumnsToInstitutionProgramsArchives < ActiveRecord::Migration[5.1]
+  def change
+    add_column :institution_programs_archives, :institution_id, :integer
```

Comment:
Hmmm, if we go this route, then this should be `institutions_archive_id`, so the association will be accurate (i.e. after archiving, it is now associated with `institutions_archives` and not `institutions`

Reply:
Right now we're expecting the schema of the archives table to match the original and we don't expect there to be a valid relationship between the institutions_programs_archives institutions_archives data.  Does that make sense?

Reply:
Then I have to ask: what is the point in archiving the `institution_programs` data if it can't be tied back to the `institutions`?

Reply:
The data is still there, but it's not a rails relationship.  Right now there isn't a method to revert institutions, but if this is something that we wanted to do, it could be done (and it would be done through SQL and this wouldn't be an issue).  This is really just a method of archiving old records because there was no system and each new version generates 30k+ rows.

Reply:
Right.  Now it also occurs to me that you can specify a different foreign key for the relationship between `InstitutionsArchive` and `InstitutionProgramsArchive`.